### PR TITLE
fix(tile-converter): skip showing progress with no analyze

### PR DESCRIPTION
**Pull request by mspivak-actionengine**

### CODE REVIEW

File: modules/tile-converter/src/i3s-converter/helpers/progress.ts [hunk 1]
```diff
@@ -119,7 +119,7 @@ export class Progress {
    * @returns Number of milliseconds remaining
    */
   getTimeRemaining(): {timeRemaining: number; trust: boolean} | null {
-    if (!this._stepsDone || !this.startTime) {
+    if (!this._stepsTotal || !this._stepsDone || !this.startTime) {
```
Comment (belom88) [inline modules/tile-converter/src/i3s-converter/helpers/progress.ts:122]: Can we add a test for this corner case?

File: modules/tile-converter/src/lib/utils/statistic-utills.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ function timeConverterFromSecondsAndMilliseconds(timeInSeconds: number, millisec
   }
 
   if (!result) {
-    result += `${milliseconds}ms`;
+    result += `${Math.floor(milliseconds)}ms`;
```
Comment (belom88) [inline modules/tile-converter/src/lib/utils/statistic-utills.ts:47]: Can we add a test for this change?

**Review by belom88 — CHANGES_REQUESTED** [review]

File: modules/tile-converter/src/lib/utils/statistic-utills.ts (hunk 2 above)
Comment (mspivak-actionengine) [reply modules/tile-converter/src/lib/utils/statistic-utills.ts:47]: Done

**Review by belom88 — APPROVED** [review]
